src/test/rustdoc/link-title-escape.rs

```diff
@@ -1,19 +0,0 @@
-// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
-// file at the top-level directory of this distribution and at
-// http://rust-lang.org/COPYRIGHT.
-//
-// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
-// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
-// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
-// option. This file may not be copied, modified, or distributed
-// except according to those terms.
-
-// compile-flags: -Z unstable-options --disable-commonmark
-
-#![crate_name = "foo"]
-
-//! hello [foo]
-//!
-//! [foo]: url 'title & <stuff> & "things"'
-
-// @has 'foo/index.html' 'title &amp; &lt;stuff&gt; &amp; &quot;things&quot;'
```

Comment:
While i know why this test was put in (a change related to intra-doc-links changed how we handled link titles in Hoedown), i don't think this test needs to be deleted? It's still something worth verifying, in case we mess with the rendering process in a similar way with Pulldown. Just take out the `compile-flags` directive, IMO.